x-pack/auditbeat/module/system/socket/helper/loopback.go

```diff
@@ -108,8 +108,7 @@ func (lo *IPv6Loopback) AddRandomAddress() (addr net.IP, err error) {
 		if err = unix.Bind(fd, &bindAddr); err == nil {
 			break
 		}
-		//nolint:errorlint // ignore
-		if errno, ok := err.(unix.Errno); !ok || errno != unix.EADDRNOTAVAIL {
+		if !errors.Is(err, unix.EADDRNOTAVAIL) {
```

Comment:
:old-man-shouting-at-clouds: x/sys/unix has two instances of %w (both in code generators rather than lib code) and AFAICS no other form of error wrapping (it would be very strange if a package that sits at this level of the stack did). While not bound by go1compat or semver, it would be extremely surprising if this ever changed since code is copied from x/sys/unix into the std tree. But if it keeps people happy…